.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE]"
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5471038)